package.json

```diff
@@ -801,27 +801,27 @@
                 },
                 {
                     "command": "aws.help",
-                    "when": "view == aws.explorer",
+                    "when": "view == aws.explorer || !aws.explorer.visible",
                     "group": "y_externalLinks@1"
                 },
                 {
                     "command": "aws.github",
-                    "when": "view == aws.explorer",
+                    "when": "view == aws.explorer || !aws.explorer.visible",
```

Comment:
how does this scale as we add other "top-level" groups?

Reply:
The idea is that, at least currently, the AWS panel is the main panel. If it's hidden, then all panels show the "general" items.

Reply:
Does this mean the AWS Toolkit menu items show up in the menus of non-AWS Toolkit panels? We had a similar bug earlier on like that.

Reply:
Fixed in https://github.com/aws/aws-toolkit-vscode/pull/1679/commits/57356fce4905a6b7adf88b15d133e72a02f3569b